Configuration/ProcessModifiers/python/phase2_hlt_vertexTrimming_cff.py

```diff
@@ -0,0 +1,6 @@
+import FWCore.ParameterSet.Config as cms
+
+# This modifier is used to enable vertex trimming in the tracking sequences
+# and in all related reconstruction modules
+phase2_hlt_vertexTrimming = cms.Modifier()
+
```

Comment:
this line is uneeded @lguzzi. Anyway please rebase to have just a single commit (also add the requested workflows).